core/config/project_settings.h

```diff
@@ -96,6 +96,7 @@ class ProjectSettings : public Object {
 	HashMap<StringName, PropertyInfo> custom_prop_info;
 	bool using_datapack = false;
 	bool project_loaded = false;
+	bool block_changed = false;
```

Comment:
I was confused by this naming as "block" has two meanings in English (and in technical English).
Could "prevent" in the name be better?

Reply:
"block" is already used in multiple similar cases (`_block_emit_changed`, `block_transform_notify`, `block_signals`, `block_minimum_size_adjust`, `block_save`, `block_update`).